### PR TITLE
size/S and larger requiring 2 LGTMs

### PR DESCRIPTION
**Pull request by bacongobbler**
Any changes labeled as `size/S` or larger should be thoroughly tested before merging and always
requires approval from 2 core maintainers. Beforehand, this only applied to `size/L` and larger.

PRs submitted by a core maintainer, regardless of size, only requires approval from one additional
maintainer. This ensures there are at least two maintainers who are aware of any significant PRs
introduced to the codebase.

Signed-off-by: Matthew Fisher <matt.fisher@microsoft.com>

### CODE REVIEW

**Review by adamreese — APPROVED** [review]

File: CONTRIBUTING.md
Comment (scottrigby) [inline CONTRIBUTING.md:235]: Just checking, didn't we want two LGTMs on all community-submitted PRs, and PRs submitted by core maintainers still only need one other LGTM? Or is that implicit here

Comment (bacongobbler) [reply CONTRIBUTING.md:235]: Yes, The next paragraph below clarifies this, though we can probably note that we mean "PRs from a community member" rather than "Any PR".

**Review by hickeyma — APPROVED** [review]
LGTM, thanks @bacongobbler

**Pull request merged**
